tradetrust/open-attestation-api/src/app.js

```diff
@@ -42,20 +43,59 @@ function create(){
     logger.error(err);
   }
 
-  app.post('/document/wrap', async function (req, res){
+  
+  app.post("/document/sign", async function(req,res){
+    console.log('/document/sign')
+    if (req.body.document === undefined){throw new UserFriendlyError('No "document" field in payload');}
+    const document = req.body.document; //should be wrapped already at this point
+
+    var keyPair;
+    if(!process.env.DOCUMENT_STORE_OWNER_PRIVATE_KEY){
```

Comment:
hardcoded default values can cause painful debugging problems. if the private key is not available in the live environment then we want this to fail noisily, not fall back to a default value and possibly be silently signing documents with the wrong private key. better to always take value from environment.

Reply:
Indeed. I'm struggling to work out the 'right' way to do this with environment variables. I think this falls under the umbrella of  changes to help running locally, for which I was imagining a seperate PR. But there's a bunch of stuff I still don't understand on that front and wanted to get feedback first. I guess I should have done a "Draft PR"?  Just noticed that option.